KS.Fiks.IO.Arkiv.Client/ForenkletArkivering/ArkivmeldingFactory.cs

```diff
@@ -81,7 +83,7 @@ public static Arkivmelding GetArkivmelding(OppdaterSaksmappe input)
                 }
                 
                 journalpost.OpprettetAv = input.sluttbrukerIdentifikator;
-                journalpost.ArkivertAv = input.sluttbrukerIdentifikator; //TODO ?????
+                journalpost.ArkivertAv = input.sluttbrukerIdentifikator; //TODO ????? Hva er egentlig problemet her?
```

Comment:
Antagelig å spesifisere bruken. Feltet vil være tomt frem til journalposten har fått status A/Arkivert. Reglene for navngiving må være som for intern korrespondansepart hvis dette skal være et felt som klienten skal kunne sette.

Reply:
Men betyr det av man burde få lov til å sette 2 forskjellige "sluttbrukeridentifikatorer" her?
Altså at input bør ha muligheten for å ha 1 for OpprettetAv og 1 for ArkivertAv?

Reply:
Vi kan ta dette på torsdag også 👍